.github/workflows/dotnet3.1Build.yml

```diff
@@ -0,0 +1,38 @@
+name: CodingGuidelines Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: 0 6 * * 1
+env:
+  version: 0.1.9
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: microsoft/setup-msbuild@v1.0.2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
```

Comment:
I think we want to update this to use .net5 even though the projects are still on 3.1